src/error.rs

```diff
@@ -555,8 +553,8 @@ pub enum WriteFailure {
     /// An error that occurred due to not being able to satisfy a write concern.
     WriteConcernError(WriteConcernError),
 
-    /// An error that occurred during a write operation that wasn't due to being unable to satisfy a
-    /// write concern.
+    /// An error that occurred during a write operation that wasn't due to being unable to satisfy
+    /// a write concern.
```

Comment:
This change looks unrelated / unnecessary, could you revert it?

Reply:
Done; automatic code formatting was a bit trigger happy here.